### DISCUSSION TITLE
이슈 템플릿

### DISCUSSION THREAD

**Discussion by jayjeong8** [EasyAndBeauty/DailyReceipt-Flutter#3] [Ideas]
### 배경
이슈를 일관된 방식으로 작성하고 쉬운 작성을 돕는 가이드를 제공하기 위해 템플릿을 만들었습니다. 
https://github.com/EasyAndBeauty/DailyReceipt-Flutter/pull/1 PR에서 작업 진행 중입니다.

관련해서 좋은 점이나 개선점 있으면 의견 부탁드립니다!

#### Feature 이슈
<img width="709" alt="image" src="https://github.com/EasyAndBeauty/DailyReceipt-Flutter/assets/82160479/2ed18307-cba5-434d-b1b0-45fef397aa3d">

#### Bug 이슈
<img width="699" alt="image" src="https://github.com/EasyAndBeauty/DailyReceipt-Flutter/assets/82160479/613f901b-58e9-4123-970a-3c81a4cb4db1">

**Comment by gygy7151** [comment]
이슈가 발생한 페이지 및 기능 뎁스를
아래와 같이 제목에 접두사에 작성하면
이슈를 파악하는데 더 빠르지 않을까 의견 올려봅니다!
ex) [메인]투두리스트>투두상세

**Reply by jayjeong8** [reply]
제목에는 생성할 때 페이지 및 기능 뎁스를 작성하는 것으로 하는 것 좋습니다👍
이슈 생성할 때 매번 값이 달라지므로 템플릿에 추가하는 대신 생성할 때 각자 이 규칙으로 입력하면 좋을 것 같아요!

feature는 Labels로 표시할 수 있도록 추가해둘게요!!

**Reply by jayjeong8** [reply]
feature Label은 기본 Labels 중 enhancement 사용하면 되겠네요!
<img width="325" alt="image" src="https://github.com/EasyAndBeauty/DailyReceipt-Flutter/assets/82160479/421aeca1-7ab2-4f53-b7d4-e3d12eee4b3f">

**Reply by gygy7151** [reply]
이슈 생성시 매번 값이 달라지는 부분까진 생각 못해봤는데 역시 제이~~ 쵝오!!!!! 좋습니댱~
